config/secrets.json.old

```diff
@@ -0,0 +1,88 @@
+{
+  "twitter": {
+    "consumer_key": "testingsdaf",
+    "consumer_secret": "test",
+    "access_token": "api",
+    "access_token_secret": "error",
+    "configured": true,
+    "on": true
+  },
+  "facebook": {
+    "accessToken": "1512246509070854|uaMaHlh6LFyu5S5ROVKmwc8tV-w",
+    "configured": true,
+    "on": true,
+    "access_token": "asfd"
+  },
+  "elmo": {
+    "authToken": "",
+    "configured": false,
+    "on": false
+  },
+  "mongodb": {
+    "username": "",
+    "password": "",
+    "host": "localhost",
+    "port": "27017",
+    "db": "aggie"
+  },
+  "email": {
+    "from": "Aggie <aggie@example.com>",
+    "transport": {
+      "method": "SES",
+      "options": {
+        "AWSAccessKeyID": "AWSAccessKey",
+        "AWSSecretKey": "AWS/Secret/Key",
+        "ServiceUrl": "https://email.us-east-1.amazonaws.com"
+      }
+    }
+  },
+  "logger": {
+    "SES": {
+      "to": "Aggie <aggie-admin@example.com>",
+      "accessKey": "AWSAccessKey",
+      "secretKey": "AWS/Secret/Key",
+      "from": "Aggie <aggie@example.com>",
+      "level": "error",
+      "silent": false
+    },
+    "file": {
+      "level": "debug",
+      "silent": false,
+      "colorize": true,
+      "maxsize": 5242880,
+      "maxFiles": 10,
+      "json": false,
+      "prettyPrint": true,
+      "filename": "logs/master.log",
+      "timestamp": true
+    },
+    "api": {
+      "log_requests": true,
+      "log_responses": true,
+      "filename": "logs/api.log"
+    },
+    "master": {
+      "filename": "logs/master.log"
+    },
+    "fetching": {
+      "filename": "logs/fetching.log"
+    },
+    "analytics": {
+      "filename": "logs/analytics.log"
+    }
+  },
+  "api_request_timeout": 60,
+  "secret": "xxx",
+  "adminParty": true,
+  "adminPassword": "letmein",
+  "fetching": false,
+  "media": {
+    "0": "t",
+    "1": "w",
+    "2": "i",
+    "3": "t",
+    "4": "t",
+    "5": "e",
+    "6": "r"
+  }
+}
```

Comment:
No! That should be the file secrets.json.example. Still needed while we don't move to a DB solution